include/srdfdom/model.h

```diff
@@ -41,7 +41,7 @@
 #include <string>
 #include <vector>
 #include <utility>
-#include <urdf/model.h>  // TODO: replace with urdf_model/types.h in Lunar
+#include <urdf/model.h>
```

Comment:
this TODO by @rhaschke should be addressed per the original PR:
https://github.com/ros-planning/srdfdom/pull/27